README.md

```diff
@@ -36,6 +36,23 @@ $ gpg-group-chat --server
 $ gpg-group-chat --client
 ```
 
+## Basic usage flow
+
+1. Server starts with gpg-group-chat --server at 192.168.0.1
+2. The client1 starts with gpg-group-chat --client --server-ip=192.168.0.1 --public\_key=public\_key1
```

Comment:
Usually is used dash instead of underscore to separate words like "--public-key"